_data/navigation.yml

```diff
@@ -38,7 +38,10 @@
         - title: Configuration File
           url: /configuration
           text: Configuration File
-        - title: Assets
+        - title: Docker Images
+          url: /images
+          text: Assets
+        - title: Docker Images
```

Comment:
I belieb you meant to adjust the text, but you replaced the "Assets" title!

Reply:
woops!